examples/client_server.jl

```diff
@@ -14,18 +14,27 @@ function echo_server(ep::UCX.Endpoint)
     data = Array{UInt8}(undef, size[1])
     wait(recv(ep, data, sizeof(data)))
     wait(send(ep, data, sizeof(data)))
-    atomic_sub!(expected_clients, 1)
 end
 
 function start_server(ch_port = Channel{Int}(1), port = default_port)
     ctx = UCX.UCXContext()
     worker = UCX.UCXWorker(ctx)
+    timer = Timer(0, interval=0.001) do t # 0.001 smallest interval
```

Comment:
@vtjnash I originally passed `0.0001` which was rounded down to `0`, therefore only firing the Timer once... 

Also passing a malformed `cb` doesn't yield an error.